src/DynamoInstall/Release.xsl

```diff
@@ -40,6 +34,14 @@
     <!--Exclude binariestosign.txt-->
     <xsl:key name="binariestosign-search" match="wix:Component[contains(wix:File/@Source, 'binariestosign.txt')]" use="@Id"/>
     <xsl:template match="wix:Component[key('binariestosign-search', @Id)]" />
+	
+	<!--Exclude RevitAddinUtility.dll-->
```

Comment:
The only Revit related files being copied by Dynamo Build process are the Sample files.